packages/tradershub/src/components/DemoRealSwitcher/DemoRealSwitcher.tsx

```diff
@@ -98,32 +56,26 @@ const DemoRealSwitcher = () => {
                 </div>
             </Button>
             {isDropdownOpen && (
-                <div className='absolute z-10 items-center w-full top-28 rounded-xs bg-system-light-primary-background shadow-10'>
+                <div className='absolute z-10 flex flex-col items-center w-full top-28 rounded-xs bg-system-light-primary-background shadow-10'>
                     {accountTypes.map(account => (
-                        <div
+                        <button
```

Comment:
what is the reason behind having `button` and `Button` ..?

Reply:
Button coming from ui has lots of base styles. Have to add unnecessary styles to overwrite it. Best to use html element button for cases like this so its just my styles applied

Reply:
@thisyahlen-deriv Why not define a variant for the Button component?

Reply:
All the variants have quite some base styles like hover. So i would need to define here to not hover and etc